yazi-config/preset/theme.toml

```diff
@@ -28,6 +28,11 @@ tab_active   = { fg = "black", bg = "lightblue" }
 tab_inactive = { fg = "white", bg = "darkgray" }
 tab_width    = 1
 
+# Selected counter
+count_selected = { fg = "white", bg = "green" }
+count_copied   = { fg = "white", bg = "yellow" }
+count_cut      = { fg = "white", bg = "red" }
```

Comment:
Its better to keep this color as same as the marker_*

Reply:
`lightred` works fine but I think light green and yellow look a bit illegible with the white text, at least with my colour scheme. Do you think it looks alright?

Reply:
Depends entirely on what those values are I suppose but you would assume the lighter colours would lean more towards white than the darker ones

Reply:
Would it be better if it were `yellow` instead of `lightyellow`? Maybe we can use a dark foreground color, like `{ fg="black", bg="yellow" }`?

Reply:
Actually yeah that might make sense as the default since the default `fg` for the active tab is `black` too, I just have that different for my config. So use light colours like the marker styles and then a black `fg`?